tests/test_directives.py

```diff
@@ -46,6 +48,19 @@ def finalize(value, dict):
     return product
 
 
+@pytest.fixture()
+def values():
```

Comment:
Either some developer documentation as to the purpose of this fixture, or a more descriptive name or both would be helpful.

Reply:
Would `non_default_directive_values` be good?